docs/user/tutorials/01-10-mtls-authentication/configure-mtls-Gardener-certs.md

```diff
@@ -0,0 +1,547 @@
+
+    ```bash
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: aws-credentials
+      namespace: test
+    type: Opaque
+    data:
+      AWS_ACCESS_KEY_ID: ...
+      AWS_SECRET_ACCESS_KEY: ...
+      # Optionally, specify the region
+      #AWS_REGION: {YOUR_SECRET_ACCESS_KEY
+      # Optionally, specify the token
+      #AWS_SESSION_TOKEN: ...
+    EOF
+    ```
+
+4. Create a DNSProvider resource that references the Secret with your DNS provider's credentials.
+   
+   See an example Secret for AWS Route 53 DNS provider:
+
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: dns.gardener.cloud/v1alpha1
+    kind: DNSProvider
+    metadata:
+      name: aws
+      namespace: default
+    spec:
+      type: aws-route53
+      ecretRef:
+        name: aws-credentials
+      domains:
+        include:
+        - "${PARENT_DOMAIN}"
+    EOF
+    ```
+
+5. Get the external access point of the `istio-ingressgateway` Service. The external access point is either stored in the ingress Gateway's **ip** field (for example, on GCP) or in the ingress Gateway's **hostname** field (for example, on AWS).
+
+    ```bash
+    LOAD_BALANCER_ADDRESS=$(kubectl get services --namespace istio-system istio-ingressgateway --output jsonpath='{.status.loadBalancer.ingress[0].ip}')
+    if [[ -z $LOAD_BALANCER_ADDRESS ]]; then
+        LOAD_BALANCER_ADDRESS=$(kubectl get services --namespace istio-system istio-ingressgateway --output jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+    fi
+    ```
+
+6. Create a DNSEntry resource.
+    
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: dns.gardener.cloud/v1alpha1
+    kind: DNSEntry
+    metadata:
+      name: dns-entry
+      namespace: test
+      annotations:
+        dns.gardener.cloud/class: garden
+    spec:
+      dnsName: "${GATEWAY_DOMAIN}"
+      ttl: 600
+      targets:
+        - "${LOAD_BALANCER_ADDRESS}"
+    EOF
+    ```
+
+7. Create the server's certificate.
+    
+    You use a Certificate resource to request and manage Let's Encrypt certificates from your Kyma cluster. When you create a Certificate, Gardener detects it and starts the process of issuing a certificate. One of Gardener's operators detects it and creates an ACME order with Let's Encrypt based on the domain names specified. Let's Encrypt is the default certificate issuer in Kyma. Let's Encrypt provides a challenge to prove that you own the specified domains. Once the challenge is completed successfully, Let's Encrypt issues the certificate. The issued certificate is stored it in a Kubernetes Secret, which name is specified in the Certificate's **secretName** field.
+
+    ```bash
+    export GATEWAY_SECRET=kyma-mtls
+    cat <<EOF | kubectl apply -f -
+    apiVersion: cert.gardener.cloud/v1alpha1
+    kind: Certificate
+    metadata:
+      name: domain-certificate
+      namespace: "istio-system"
+    spec:
+      secretName: "${GATEWAY_SECRET}"
```

Comment:
The GATEWAY_SECRET value is hard coded in line 139. Having this as an env honestly had me confused of where did it suddenly come from (I'm probably blind as I did not expect this bash to do anything but create this resource 😉)
Maybe we could just put the value "kyma-mtls" everywhere instead of this var ${}? Var usage here IMHO blurs the visibility of what is actually configured

Reply:
Or it should be an explicit step that says something like: please choose a name for this secret, and run export GATEWAY_SECRET=<yourname>